### DISCUSSION TITLE
家里的服务器怎么自启动

### DISCUSSION THREAD

**Discussion by FengNewp** [natfrp/wiki#378] [使用问题]
Linux：  centos 7.9
软件版本：0.45.0-sakura-7
把frpc放在了/home的目录下，配置文件frpc.ini一同放在了目录下
怎么后台自启动frpc -c frpc.ini
frpc -c frpc.ini & 这个没有用

**Comment by fakeboboliu** [comment]
请按照 https://doc.natfrp.com/launcher/usage.html 配置，自然就可以自启了
